### ISSUE TITLE
Error 404 sending to remote server

### ISSUE THREAD

**Issue by dh47** [pagermon/pagermon#223]
I have setup my pi as the client and I have a remote connection to the server side, all is well but I'm having issues sending the data to the remote server.

https://pastebin.com/skYsq0wk

Thanks.

**Comment by davidmckenzie** [comment]
What's your `hostname` set to in your client's config file?

**Comment by marshyonline** [comment]
Looks to me that it's hitting the home page and not the API 
You need to make sure that HOSTNAME is the client config is set to 
```http://your-ip-or-domain/api/messages

```

**Comment by dh47** [comment]
"hostname": "http://XX.63.26.185:3000/",

**Comment by DanrwAU** [comment]
As per Marshy's comment (which i just fixed lol) you need to put the full path to the api in there. 

so it would be `"hostname": "http://XX.63.26.185:3000/api/messages"`

**Comment by dh47** [comment]
Wow, things that can happen when you add a "/" at the end.

**Comment by davidmckenzie** [comment]
Yeaaah that shouldn't happen... it should be smart enough to deal with that :D Will reopen and fix!

**Comment by DanrwAU** [comment]
Should be able to do soemthing like this

```
function stripTrailingSlash(hostname) {
    if(hostname.substr(-1) === '/') {
        return hostname.substr(0, hostname.length - 1);
    }
    return hostname;
}
```

**Comment by dh47** [comment]
Just wanted an update on this?  Has this been pushed?

**Comment by DanrwAU** [comment]
I haven't seen a commit for it yet. 

Unfortunately @davidmckenzie @marshyonline and myself have all been busy with life stuff recently. Personally i haven't really had a chance to do much of anything with pagermon recently.

**Comment by marshyonline** [comment]
I don't think we have pushed a fix to strip the tailing slash
Quick way to fix on your end is to remove the trailing slash from the hostname

**Comment by dh47** [comment]
Okay.

**Comment by DanrwAU** [comment]
This is fixed in https://github.com/pagermon/pagermon/pull/233

**Comment by DanrwAU** [comment]
This will be fixed in v0.3.0 ... closing 😁
